logics.py

```diff
@@ -7,7 +7,7 @@
 
 __author__ = "Jan Max Meyer"
 __copyright__ = "Copyright 2015-2019 by Mausbrand Informationssysteme GmbH"
-__version__ = "2.4.1"
+__version__ = "2.5"
```

Comment:
I think we should consider the typically version-number-triple (Major.Minor.Patch).
```suggestion
__version__ = "2.5.0"
```